src/main/java/com/xjeffrose/chicago/DBManager.java

```diff
@@ -145,7 +155,9 @@ boolean write(byte[] colFam, byte[] key, byte[] value) {
       log.error("Tried to write a null value");
       return false;
     } else if (!colFamilyExists(colFam)) {
-      createColumnFamily(colFam);
+      synchronized (columnFamilies) {
```

Comment:
columnFamilies is already concurrent. It doesn't need to be synchronized. This just adds and additional unnecessary lock.